tasks/get_command_output.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+require 'open3'
```

Comment:
Lets use https://github.com/chef/mixlib-shellout or 

```
backend = Train.create('local', target_config)
# start or reuse a connection
conn = backend.connection
# loop around commands
cmd = conn.run_command('inspec version')
cmd.stdout
conn.close
```
under the hood, train is using mixlib-shellout